docs/Tasks.txt

```diff
@@ -104,4 +105,9 @@ SYSTEMDS-150 Releases
  
 SYSTEMDS-160 Tensor Compiler/Runtime
  * 161 Local readers and writers for tensors (all formats)
- * 162 Spark binary tensor instructions
\ No newline at end of file
+ * 162 Spark binary tensor instructions
+ 
+ SYSTEMDS-170 Partial Reuse and Experiment
+ * 171 Initial version of partial rewrites
+ * 171 Rewrites for rbind-tsmm, cbind/rbind-elementwise */+
```

Comment:
please try to stick to one line per task and split the tasks into reasonable-sized chunks that can be addressed in few days each.